src/test/java/io/personium/core/rs/cell/TokenEndPointResourceTest.java

```diff
@@ -79,26 +77,26 @@
 import io.personium.test.categories.Unit;
 
 /**
- * TokenEndPointResource unit test classs.
+ * TokenEndPointResource unit test class.
  */
-@RunWith(PowerMockRunner.class)
-@PrepareForTest({ TokenEndPointResource.class, ResidentRefreshToken.class, VisitorRefreshToken.class,
-    AbstractOAuth2Token.class })
 @Category({ Unit.class })
-@PowerMockIgnore({"javax.crypto.*" })
 public class TokenEndPointResourceTest {
+    static Logger log = LoggerFactory.getLogger(TokenEndPointResourceTest.class);
+
 
     /** Target class of unit test. */
     private TokenEndPointResource tokenEndPointResource;
     private Cell mockCell;
     private CellRsCmp mockCellRsCmp;
     private UriInfo mockUriInfo;
+    private String xForwadedFor = "1.2.3.4";
 
     @BeforeClass
-    public static void beforeClass() {
+    public static void beforeClass() throws Exception {
         PersoniumCoreApplication.loadConfig();
+        TransCellAccessToken.configureX509(PersoniumUnitConfig.getX509PrivateKey(),
+                PersoniumUnitConfig.getX509Certificate(), PersoniumUnitConfig.getX509RootCertificate());
```

Comment:
Necessary initialization (loading X509 files) for using TransCellAccessToke class in the tests.